LiskHQ/op-fault-detector
# Add prometheus metric

### What was the problem?

This PR resolves #9 

### How was it solved?

- Create prometheus registry and initialize it in the app
- Create FaultDetectorMetrics and add metric needed for fault detector
- Expose metric from server

### How was it tested?

- Run `go run ./cmd`
- Check for metrics http://localhost:8080/metrics
